README.md

```diff
@@ -13,5 +16,14 @@ the function and run automatically when you commit through a GitHub action.
 
 ## Runtime Analysis
 
-What is the worst-case big $\Theta$ complexity of your implementation? Add your
-answer, including your reasoning, to this markdown file.
+What is the worst-case big $\Theta$ complexity of your implementation? 
+
+The worst case big $\Theta$ seems like it should be $\Theta(V^4)$  with V
+being the number of vertices in the graph. This is because we get $\Theta(V)$ 
+from the first loop in the hasCycle() function because we iterate over all of
+the vertices. Then we use another loop for checking for any edges between each
+vertex which gives us another $\Theta(V)$. Then we have .includes() which iterates
+over all of the vertices that we have already covered which gives us another $\Theta(V)$.
+Then finally the recursions worst case complexity would be to iterate over each vertex
```

Comment:
I don't see where the last $|V|$ is coming from -- didn't you account for this already when you analyzed the complexity of the helper function?